migrations/20200713233601-add-hospital-column-to-patients.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.sequelize.transaction(t => {
+      return Promise.all([
+        queryInterface.addColumn(
+          "patients",
+          "hospitalId",
+          {
+            type: Sequelize.DataTypes.INTEGER
+          },
+          { transaction: t }
+        )
+      ]);
+    });
```

Comment:
sort of a nit - if there's only one schema change being made, there's no need to wrap it in a transaction; would like to see this change for consistency
```suggestion
    return queryInterface.addColumn("patients", "hospitalId", { type: Sequelize.DataTypes.INTEGER });
```